### PR TITLE
Assignment- 5 Completed

### PR DESCRIPTION
**Pull request by HarsH-singHh**
Screenshot (832)

#------------------------------------------------------------------------------------------------------#

Screenshot (833)

### CODE REVIEW

**Comment by Nocturnal22** [comment]
Dog website : fb/twitter/insta/linkedin clickbaits not working.

File: assignment5.html [hunk 1]
```diff
@@ -12,33 +12,52 @@
...
+			<a href="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1145795/plant-4.png" target="_blank" class="btn">I'LL Take
+				10</a>
 		</section>
-	</div>	
+		<!-- Section2 ends -->
 
+	</div>
+	<!-- Div(card) ends -->
 
 	<!-- End your code -->
```
Comment (TannuGARG) [inline assignment5.html:63]: Correct

**Review by TannuGARG — COMMENTED** [review]
@HarshS15j 
correct!

**Comment by saikiransathpadi** [comment]
layout is correct, but the image positions are not proper, the image section should be scaled properly, (check the edges placement in the design)

**Comment by HarsH-singHh** [comment]
> layout is correct, but the image positions are not proper, the image section should be scaled properly, (check the edges placement in the design)

how to achieve that? not very familiar with the terminology edges placement.

**Pull request merged**